setup.py

```diff
@@ -207,6 +231,8 @@ def setup_package():
 
         if not is_source_release(root):
             generate_cython(root, "spacy")
+            print("COnverting language data to msgpacks...")
+            binarize_jsonl_data(root, JSONL_DATA_TO_CONVERT)
```

Comment:
Saving polish data to rookt will create problems like name conflicts with other languages, I suggest creating a folder for saved msgpacks and refactor the configuration accordingly.

Reply:
Another, less elegant but simpler solution, would be to squash path of saved file into a prefix (for example by replacing "/" with "_"

Reply:
Unless `filename` in `binarize_jsonl_data` is actually a file path, if this is the case than there is no problem with conflicts, but please rename it (in all places where it is a path instead of a name) - it's extremely confusing

Reply:
Changed ``filename`` -> ``filepath``